cluster-autoscaler/cloudprovider/packet/packet_node_group_test.go

```diff
@@ -29,36 +29,36 @@ import (
 	"github.com/stretchr/testify/mock"
 )
 
-const createPacketDeviceResponsePool2 = ``
-const deletePacketDeviceResponsePool2 = ``
+const createMetalDeviceResponsePool2 = ``
+const deleteMetalDeviceResponsePool2 = ``
 
-const createPacketDeviceResponsePool3 = ``
-const deletePacketDeviceResponsePool3 = ``
+const createMetalDeviceResponsePool3 = ``
+const deleteMetalDeviceResponsePool3 = ``
 
 func TestIncreaseDecreaseSize(t *testing.T) {
-	var m *packetManagerRest
+	var m *equinixMetalManagerRest
 	server := NewHttpServerMock(MockFieldContentType, MockFieldResponse)
 	defer server.Close()
 	assert.Equal(t, true, true)
-	if len(os.Getenv("PACKET_AUTH_TOKEN")) > 0 {
+	if len(os.Getenv("PACKET_AUTH_TOKEN")) > 0 || len(os.Getenv(metalAuthTokenEnv)) > 0 {
 		// If auth token set in env, hit the actual Packet API
-		m = newTestPacketManagerRest(t, "https://api.packet.net")
+		m = newTestMetalManagerRest(t, "https://api.equinix.com")
```

Comment:
Should this be api.equinix.com/metal/v1 as it was in other `newTestMetalManagerRest` calls

Reply:
It probably doesn't really change the mocks, but it would _look_ right.

Reply:
Ah yes, missed this one. Fixed